home-clients/pom.xml

```diff
@@ -72,8 +72,6 @@
                             <generateApiTests>false</generateApiTests>
                             <generateModelTests>false</generateModelTests>
                             <configOptions>
-                                <java8>true</java8>
```

Comment:
Why did you remove it? Can it affect anything?

Reply:
Well, old team said that generator doesn't work with Java 11 so they had to set java 8. But now we changed generator and it works with new java so we don't have to it anymore.

Reply:
Done.